File: Day-18/vd4.py
# Cho một xâu chứa ngày tháng dạng d/m/y, in ra từng thành phần d, m, y
st = '8/6/2022'
ds = st.split('/')
(ngay, thang, nam) = ds # unpack
print(f'Ngày {ngay}, tháng {thang}, năm {nam}')

File: Day-18/vd4.js
// Cho một xâu chứa ngày tháng dạng d/m/y, in ra từng thành phần d, m, y
let st = '8/6/2022';
let ds = st.split('/');
let [ngay,thang,nam]=ds;
console.log(`Ngày ${ngay}, tháng ${thang}, năm ${nam}`)